client/NavBar.jsx

```diff
@@ -28,40 +28,71 @@ class InnerNavBar extends React.Component {
     }
 
     renderMenuItem(menuItem) {
+        let active = menuItem.path === this.props.path ? 'active' : '';
+        let title = this.props.user ? menuItem.title.replace('%CURRENTUSER%', this.props.user.username) : menuItem.title;
```

Comment:
This replacement seems slightly strange since we're also already special casing the only time it occurs (`showProfilePic`). It's fine, but it introduces 2 different independent keying conditions that are unlikely to ever be used independently.